src/ui/index.ts

```diff
@@ -23,3 +23,4 @@ export * from './tag';
 export * from './tags';
 export * from './textarea-field';
 export * from './tree';
+export * from './layout';
```

Comment:
Alph order

Reply:
I think this is not alph order... 
![image](https://user-images.githubusercontent.com/34678770/97553016-3867fe80-19d5-11eb-9f8e-386a9d2551c2.png)

Reply:
Should be aplha